.github/workflows/snyk.yaml

```diff
@@ -6,6 +6,11 @@ on:
     branches:
       - master
       - release-*
+  pull_request:
+    branches:
+      - master
+      - release-*
```

Comment:
I think it's probably good to keep running these checks in PRs. This could help us catch license changes like this one when dependabot does its updates. @travisn wdyt?